### DISCUSSION TITLE
Fehlerhafter Backup nach Installation der neuesten Version 3.67.10.20230114

### DISCUSSION THREAD

**Discussion by pekae** [OpenCCU/OpenCCU#2210] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

mit der aktuellen 3.67.10.20230114 wird der Backup verstümmelt abgelegt. Zuvor war das Backup über 300kb groß, nun werden nur 10 Kb gesichert. Beim Versuch das Backup wieder einzuspielen kommt Fehlermeldung - Backup ungültig
Die Backupdateien haben die verringerte Größe bei der automatischen Sicherung (Nachts) und bei der Sicherung auf den PC

### Describe the behavior you expected

Backup erstellen und ggfs wieder einspielen können

### Steps to reproduce the issue

1.  Backup erstellen
2. Backup versucht einzuspielen
3.
...

### What is the version this bug report is based on?

3.67.10.20230114

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
keine relevanten Einträge im Log
```

**Comment by jens-maus** [comment]
Da dies kein generellen Bug/Issue Charakter hat habe ich das mal in ein Diskussionsbeitrag verwandelt weil du ja nach Hilfe für dein spezielles Problem suchst.

Bitte erst einmal genau erläutern wie du das Backup generieren lässt? Vmtl. auf einen angeschlossenen USB Stick. Wenn ja, dann wird dieser vmtl. einfach defekt sein.

**Reply by pekae** [reply]
wie beschrieben, sichere ich sowohl auf den Stick, als auch auf den PC

**Reply by pekae** [reply]
genauer : ssd und PC
die Dateiensind immer exact gleich groß

**Reply by jens-maus** [reply]
Dann müsstest du erst einmal genauer ausführen was genau im Backup fehlt. Selbst auspacken, reinschauen, usw. Und mal via SSH einloggen und z.B. `createBackup.sh` aufrufen und testen.

**Reply by pekae** [reply]
createBackup.sh hab ich schon ausgeführt - ohne Meldungen, gleiche Größe

ich entpacke mal und schaue

**Reply by pekae** [reply]
usr_local.tar.gz
signature
signature.sha256
key_index
firmware_version

dies ist der Inhalt

**Reply by jens-maus** [reply]
> dies ist der Inhalt

Das ist schon klar. Die Frage ist nur was steckt in der `usr_local.tar.gz`. Offensichtlich geht das Backup ja wohl ohne fehler durch. D.h. es muss irgendwas an der Quelle (in deinem live system) ggf. nicht passen.

**Reply by pekae** [reply]
.nobackup

was mach ich falsch ?

**Reply by jens-maus** [reply]
Wirklich? Nur eine `.nobackup` Datei? Dann hast du ganz offensichtlich gedacht es ist eine gute Idee diese Datei im root des `/usr/local` abzulegen und hast damit den Backupskript angewiesen keinerlei Dateien unterhalb dieses verzeichnisbaumes (das gesamte `/usr/local`) nicht zu backupen.

**Reply by pekae** [reply]
wenn das so ist, nicht wissentlich - ich schaue

**Reply by pekae** [reply]
tatsächlich - ich wars nicht :-)

**Comment by jens-maus** [comment, marked as answer]
Ok, dann halten wir das mal für die Nachwelt fest. Du hast fälschlicherweise eine `/usr/local/.nobackup` Datei angelegt und dir damit selbst ins Bein geschossen, denn alle Verzeichnisse unterhalb des Verzeichnisses in der die `.nobackup` Datei liegt wird dann nicht vom Backup erfasst.

Und das folgende SSH Kommando sollte das Problem entsprechend bereinigen:

`rm -f /usr/local/.nobackup`

Danach sollten dann die Backups wieder normal sein.

**Reply by pekae** [reply]
wie gesagt, nicht wissentlich
ich habe das System auf einer SSD neu installiert und dann das Backup der SD-Karte eingespielt. 

Sorry, für die unnötige Mühe und herzlichen Dank für die prompte Hilfe

**Reply by pekae** [reply]
ich hätte noch eine Frage zum mount

ich habe in der fstab einen mount Befehl reingeschrieben
dieser funktioniert auch.

nach dem Reboot ist der Eintrag in der fstab aber wieder gelöscht - wieso ?
wie kann ich das verhindern

**Reply by jens-maus** [reply]
Kein Thread hijacking bitte. Das Thema hier ist abgeschlossen. Wenn du neue/andere Fragen hast, neuen Diskussionsbeitrag eröffnen.

**Comment by Baxxy13** [comment]
Mysteriös wo die `/usr/local/.nobackup` herkommt.

Spekulativ würde ich sagen da wurde der Alternative Speicherpfad (CustomStoragePath) über die "Erweiterten Systemeinstellungen" auf `/usr/local` gesetzt.

Anders lässt sich das wohl nicht erklären...
